### ISSUE TITLE
unreasonable：the loss is optimized to negative value

### ISSUE THREAD

**Issue by xueqings** [xmed-lab/UCVME#1]
<img width="620" alt="损失为负值" src="https://github.com/user-attachments/assets/a9f8ffd6-298b-42d5-8b06-aa5f925250d2">
According to the derivation of your supplementary materials, the heteroscedasticity loss should be positive, and the total loss should also be positive. However, in the process of code training, the loss is optimized to negative value, and I did not find an obvious lower bound. May I ask if it is a problem set by me in the process of code training?May I ask the author to explain why the loss is negative? Thank you very much!

**Comment by xueqings** [comment]
> The loss function is stated in equation 1 of the paper and is the average of
> 
> ( (yi - y)^2 ) / 2Var + ln(Var)/2
> 
> When Var < 1, ln(Var) < 0, and this can be larger than the first term, and this is easily the case if the labels have been standardized to 0 mean unit standard deviation

![image](https://github.com/user-attachments/assets/1597e4b4-2990-45bc-ab7d-aaaa9bd9c2f5)
Thank you very much for your answer. I still have some questions.：① In the author's supplementary proof, the objective function is derived by negative log-likelihood, looking at equations (4) and (5), it is obvious that the value of the objective function cannot be negative; ② In addition, if the loss function is negative, it should also have a lower bound. Otherwise, isn't the parameter optimal when the loss value is negative infinity? I didn't find a clear lower bound in the code either. f you are capable of answering my questions, I would be extremely grateful.
